src/test/java/rx/internal/operators/OperatorDoOnRequestTest.java

```diff
@@ -76,5 +73,55 @@ public void onNext(Integer t) {
                 });
         assertEquals(Arrays.asList(3L,1L,2L,3L,4L,5L), requests);
     }
+    
+    @Test
+    public void dontRequestIfDownstreamRequestsLate() {
```

Comment:
Yay! You did it! `@Test` without `test` prefix, oh, I'm so happy now :smile_cat: